modules/logging-loki-gui-install.adoc

```diff
@@ -0,0 +1,87 @@
+// Module is included in the following assemblies:
+// 
+:_content-type: PROCEDURE
+[id="logging-loki-gui-install_{context}"]
+= Installing logging Operators using the {product-title} web console
+
+To install and configure logging on your {product-title} cluster, additional Operators must be installed. This can be done from the Operator Hub within the web console.
+
+{Product-title} Operators use custom resources (CR) to manage applications and their components. High-level configuration and settings are provided by the user within a CR. The Operator translates high-level directives into low-level actions, based on best practices embedded within the Operator’s logic. A custom resource definition (CRD) defines a CR and lists all the configurations available to users of the Operator. Installing an Operator creates the CRDs, which are then used to generate CRs.
+
+.Prerequisites
+
+* Supported object store (AWS S3, Google Cloud Storage, Azure, Swift, Minio, OpenShift Data Foundation)
+
+.Procedure
+
+. Install the `Loki Operator`:
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Type `Loki Operator` in the filter by keyword box. Choose *Loki Operator* from the list of available Operators and click *Install*.
++
+[NOTE]
+====
+The Community Loki Operator is not supported by Red Hat.
+====
+
+.. On the Install Operator page, for *Update Channel* select *stable*.
++
+--
+include::snippets/logging-stable-updates-snip.adoc[]
+--
+
+As the Loki Operator must be deployed to the global operator group namespace `openshift-operators-redhat`, *Installation mode* and *Installed Namespace* is already be selected. If this namespace does not already exist, it is created for you.
+
+.. Select *Enable operator-recommended cluster monitoring on this namespace.*
+
+This option sets the `openshift.io/cluster-monitoring: "true"` label in the Namespace object. You must select this option to ensure that cluster monitoring scrapes the `openshift-operators-redhat` namespace.
+
+.. For *Update approval* select *Automatic*, then click *Install*.
++
+--
+include::snippets/logging-approval-strategy-snip.adoc[]
+--
+
+.Procedure
```

Comment:
There should only be one procedure per module. Per the mod docs guidelines, "A procedure module contains numbered, step-by-step instructions to help the user accomplish a single task."